### PR TITLE
FE-1413 help component

### PR DESCRIPTION
**Pull request by grabkowski**
- Moved styles from SCSS to styled-component,
- Added DLS styling,

### CODE REVIEW

File: src/components/help/help.style.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    cursor: pointer;
+    text-decoration: none;
+
+    &:hover {
+      color: $blue;
+      text-decoration: underline;
+    }
+  `}
+
+  ${({ theme }) => theme.name === THEMES.classic && css`
```
Comment (izakrzewska) [inline src/components/help/help.style.js]: isClassic helper can be used here

File: src/components/help/help.component.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        tooltipPosition={ tooltipPosition }
+        tooltipAlign={ tooltipAlign }
+      />
+    </StyledHelp>
+  );
+};
+
+Help.propTypes = {
+  /** A custom class name for the component. */
+  className: PropTypes.string,
```
Comment (dawidzarzycki) [inline src/components/help/help.component.js]: should we add information about legacy prop?

**Comment by harpalsingh** [comment]
Should all configs in the knobs work?

Using tooltipPosition: right and tooltipAlign: right, we get the following visual (same for left, but top and bottom position work fine with those alignments)

Screenshot 2019-07-16 at 10 56 05

**Comment by grabkowski** [comment]
> Should all configs in the knobs work?
> 
> Using tooltipPosition: right and tooltipAlign: right, we get the following visual (same for left, but top and bottom position work fine with those alignments)
> 
> Screenshot 2019-07-16 at 10 56 05

The Component behaved the same way in the previous version so I assume that this is the correct behaviour.

**Comment by harpalsingh** [comment]
@grabkowski the problem is more to do with left/right alignment would never really need any additional config, so its more of a config mix issue, so not a problem.

**Pull request merged**